### PR TITLE
Fix installation links

### PR DESCRIPTION
**Pull request by pgressa**

### CODE REVIEW

File: buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy [hunk 1]
```diff
@@ -29,10 +29,9 @@ class DownloadPage {
         new GuideGroup(title: "Install",
                 image: "${getImageAssetPreffix(url)}download.svg",
                 items: [
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installSdkman", title: 'Install with SDKman'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installHomebrew", title: 'Install with Homebrew'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installMacPorts", title: 'Install with MacPorts'),
-                        new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#installWindows", title: 'Install through Binary on Windows'),
+                        new GuideGroupItem(href: "https://micronaut-projects.github.io/micronaut-starter/latest/guide/#installation", title: 'Install with SDKman'),
```
Comment (pgressa) [inline buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy:32]: there are no anchors, should I create anchors for these aka, make them <h>'s?

File: buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy [hunk 2]
```diff
@@ -29,10 +29,9 @@ class DownloadPage {
...
+                        new GuideGroupItem(href: "https://micronaut-projects.github.io/micronaut-starter/latest/guide/#installation", title: 'Install with Homebrew'),
+                        new GuideGroupItem(href: "https://micronaut-projects.github.io/micronaut-starter/latest/guide/#installation", title: 'Install through Chocolatey on Windows'),
                         new GuideGroupItem(href: "https://docs.micronaut.io/latest/guide/index.html#buildSource", title: 'Building from Source'),
```
Comment (pgressa) [inline buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy:35]: Current documentation doesn't contain anything related to build from source, so IMHO that should be fixed as well

File: buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy (hunk 1 above)
Comment (sdelamo) [reply buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy:32]: I think we should split installation in starter into multiple sections.

File: buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy (hunk 2 above)
Comment (sdelamo) [reply buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy:35]: Is the "build from source" complete gone? It does not exist in `core` docs or in `starter`?

File: buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy (hunk 1 above)
Comment (sdelamo) [reply buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy:32]: I have created the PR and assign it to you: https://github.com/micronaut-projects/micronaut-starter/pull/471

File: buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy (hunk 2 above)
Comment (pgressa) [reply buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy:35]: @sdelamo didn't find at all

Comment (sdelamo) [reply buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy:35]: I would say remove we need to remove the menu option or add the documentation back.

Comment (sdelamo) [reply buildSrc/src/main/groovy/io/micronaut/download/DownloadPage.groovy:35]: Here is the [Build from source](https://docs.micronaut.io/1.3.7/guide/index.html#buildSource) documentation snippet in 1.3.7

**Comment by sdelamo** [comment]
Closing this PR after @JasonTypesCodes commit: https://github.com/micronaut-projects/static-website/commit/2d7b2baeb616ee22d72aaf4ad3f79dd9f066074d

If I am missing anything, please let me know.

**Pull request closed without merging**